index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Build Survey Form With CSS</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  
+      <h1 id="title">Microverse Trial Collaboration Survey Form</h1>
+      <form id="form">
+        <div class="form-control">
+          <label for="name" id="name-label">
+            Name
+          </label>
+          <input type="text"
+                  id="name"
+                  placeholder="Enter your name" />
+        </div>
+        <div class="form-control">
+             <label for="email" id="email-label">
+                   Email
+             </label>
+             <input type="email"
+             id="email"
+             placeholder="Enter your email" />
+        </div>
+
+        <div class="form-control">
+           <label for="age" id="age-label">
+               Age
+           </label>
+
+           <input type="text"
+            id="age"
+            placeholder="Enter your age" />
+        </div>
+
+        <div class="form-control">
+           <label for="role" id="role-label">
+                 Which option best describes you?
+           </label>
+
+
+           <select name="role" id="role">
+               <option value="student">Student</option>
+               <option value="intern">Intern</option>
+               <option value="professional">
+                   Professional
+               </option>
+               <option value="other">Other</option>
+               </select>
+          </div>
+
+          <div class="form-control">
+               <label>
+                   Would you recommed Microverse to a friend?
+               </label>
```

Comment:
Label tags are best used for inputs. As this is not an input, I would suggest using a <h4> or <h5> or any other appropriate tag.